src/foam/u2/wizard/data/CreateLoader.js

```diff
@@ -54,7 +54,7 @@ foam.CLASS({
       if ( this.delegate ) {
         const delegateResult = await this.delegate.load(o);
         if ( delegateResult ) {
-          return delegateResult.copyFrom(this.args);
+          return this.updateWithSpec ? delegateResult.copyFrom(this.args) : delegateResult;
```

Comment:
makes sense but any idea if this will affect anything? asking because comment above
`// If CreateLoader has a delegate we assume copyFrom is expected`

Reply:
Seems reasonable, especially since it doesnt impact any current behaviour.

Reply:
In it's current state it will lose and data updated by the user if returning to a wizardlet